builds/e2e/templates/e2e-setup.yaml

```diff
@@ -151,7 +151,8 @@ steps:
       rootCaCertificatePath = "$rootCaCertificatePath";
       rootCaPrivateKeyPath = "$rootCaPrivateKeyPath";
       logFile = Join-Path '$(binDir)' 'testoutput.log';
-      verbose = '$(verbose)'
+      verbose = '$(verbose)';
+      uploadsupportbundle = 'true'
```

Comment:
This new key should be camel-cased, e.g. "uploadSupportBundle", to be consistent with all the other keys in context.json.

Reply:
Also, the test code is not uploading the support bundle, it is just getting it. So maybe, "supportBundle"?